cmd/osv-scanner/main.go

```diff
@@ -99,14 +99,25 @@ func run(args []string, stdout, stderr io.Writer) int {
 			},
 			&cli.BoolFlag{
 				Name:  "experimental-call-analysis",
-				Usage: "attempt call analysis on code to detect only active vulnerabilities",
+				Usage: "[Deprecated] attempt call analysis on code to detect only active vulnerabilities",
 				Value: false,
+				Action: func(context *cli.Context, b bool) error {
```

Comment:
Does this keep the behavior the same? or just return an error?

I think we want to emulate the previous behavior (basically, if this flag is passed in, pretend --call-analysis=all is passed in), and print a warning message.

Reply:
Updated. Thanks Rex!